integration/single-node-slug-deploy/check.go

```diff
@@ -300,7 +301,6 @@ func postHelloManifest(dir string) error {
 	manifest := &pods.Manifest{}
 	manifest.Id = "hello"
 	manifest.StatusPort = 43770
-	manifest.StatusHTTP = true
```

Comment:
The tests and the watch pkg were both backwards. Check was listening on https and when statusHTTP was true the watch pkg was using https